lib/chef-dk/command/shell_init.rb

```diff
@@ -106,7 +106,7 @@ def run(argv)
           export(shell_name, var_name, value)
         end
 
-        msg(completion_for(shell_name))
+        msg(completion_for(shell_name)) unless shell_name == "powershell"
```

Comment:
Probably keep it in a local variable so we don't do the work twice, e.g.,

```ruby
completion_commands = completion_for(shell_name)
msg(completion_commands) unless completion_commands.empty?
```

You could go further and make a method like

```ruby
def emit_shell_cmd(cmd)
  msg(cmd) unless cmd.empty?
end
```

Either one is probably okay.